data.sql

```diff
@@ -44,3 +44,25 @@ UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon','Pikachu');
 UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon','Plantmon');
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander','Squirtle','Blossom');
 UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon','Boarmon');
```

Comment:
This `UPDATE` are changing the table so from line 38 - 46 should have to be moved in to the `schema.sql` file. Kindly, move those to keep the file structure professional.